src/types/ApplicationFormField.ts

```diff
@@ -29,11 +28,8 @@ const writableApplicationFormFieldWithApplicationContextSchema: JSONSchemaType<W
 			position: {
 				type: 'integer',
 			},
-			label: {
-				type: 'string',
-			},
 		},
-		required: ['baseFieldId', 'position', 'label'],
+		required: ['baseFieldId', 'position'],
```

Comment:
Hmm -- we shouldn't be removing label from application form field.  That's its own concept.

Base fields are PDC specific, and are therefore localized.  Application form fields reflect the labels in the funder's GMS.

All this is to say: we should keep this concept as is; we won't be localizing it, but we aren't removing support for it either.